docs/core/tools/dotnet-run.md

```diff
@@ -60,7 +60,10 @@ Builds and runs the app using the specified [framework](../../standard/framework
 
 `-p|--project <PATH>`
 
-Specifies the path to the project file. It can be a path to a [csproj](csproj.md) file or to a directory containing a [csproj](csproj.md) file. It defaults to current directory if not specified. 
+Specifies the path to the project file. It can be a path to a [csproj](csproj.md) file ~or to a directory containing a [csproj](csproj.md) file~ (See NOTE). It defaults to current directory if not specified.
+
+> [!NOTE]
+> Use the full path to the project file with the `-p|--project` option. A regression in the CLI prevents providing a folder path at this time. For more information and to track this issue, see [dotnet run -p, can not start a project (dotnet/cli #5992)](https://github.com/dotnet/cli/issues/5992).
```

Comment:
The full path implies that relative paths aren't supported. I'd change this to "Use the path and name of the project file..."